tests/emulator/src/vbridge_impl.cc

```diff
@@ -155,11 +169,18 @@ void VBridgeImpl::run() {
     // spike =======> to_rtl_queue =======> rtl
     loop_until_se_queue_full();
 
+    bool exit_after_all_issued = false;
     // loop while there exists unissued insn in queue
     while (!to_rtl_queue.front().is_issued) {
       // in the RTL thread, for each RTL cycle, valid signals should be checked, generate events, let testbench be able
       // to check the correctness of RTL behavior, benchmark performance signals.
       SpikeEvent *se_to_issue = find_se_to_issue();
+      if (se_to_issue->is_exit_insn) {
+        se_to_issue->is_issued = true;
+        exit_after_all_issued = true;
+        continue;
```

Comment:
Maybe we should prevent instructions being issued after exit instruction?

Reply:
The exit instruction is guaranteed to be the last one in the queue.

Reply:
Agreed. Is it necessary to log the exit message? If it's not, we can return from here.

Reply:
Agreed. Is it necessary to log the exit message? If it's not, we can return from here.

Reply:
edited

Reply:
log then exit seems fine